### ISSUE TITLE
invalid multipart error

### ISSUE THREAD

**Issue by jbenet** [ipfs/go-ipfs#2172]
I've been seeing this occasionally, sometimes right after booting the daemon:

```
# daemon
12:45:41.707 ERROR commands/h: err: multipart: NextPart: multipart: Part Read: invalid byte in chunk length handler.go:270

# client
12:45:41.709 ERROR commands/h: multipart: NextPart: multipart: Part Read: invalid byte in chunk length client.go:271
```

ipfs version 0.3.11-dev

yay more multipart problems

cc @whyrusleeping

**Comment by jbenet** [comment]
(add fails)

**Comment by Dignifiedquire** [comment]
I thought this was fixed with go@1.5.2 in 4.0?

**Comment by jbenet** [comment]
@Dignifiedquire ah maybe -- i'll close and reopen if i see it again on 0.4.0
